src/EditorFeatures/CSharpTest/AddParameter/AddParameterTests.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
```

Comment:
I haven't reviewed this in depth yet.  But can you verify that you have a bunch of tests around the cascading option?  That includes tests where all the changes are in one file, vs at least a few tests where the changes cross into different files.

ideally, we'd have cross language tests as well, but those are harder to write.

Reply:
Test coverage isn't complete. For cascading I do have 

* partials in same file and across files
* virtual, override across files and projects including coming from metadata
* explicit and implicit interfaces

I don't have written any VB tests and therefore cross language is missing too. That's my biggest todo right now (I had forgotten to mention this earlier)

Reply:
No worries.  Will want that for completeness.  But no rush :)